modules/ROOT/pages/create-map-json-xml-to-outbound-x12.adoc

```diff
@@ -1,19 +1,19 @@
-= Create a Map For XML or JSON to Outbound EDI X12
+= Create a Map For XML or JSON to Outbound X12
 
 Create a template project in Anypoint Studio to import a template that creates a DataWeave map to transform application formats such as XML and JSON to outbound EDI X12 transactions, then test the map locally.
```

Comment:
Don't know if this matters, but you removed EDI in the title but it's still referred to as "outbound EDI X12..." on line 3.

Reply:
I am planning to remove EDI from titles to save wording, but will have it on first mention in a topic as I update it.